RELEASE.rst

```diff
@@ -0,0 +1,27 @@
+RELEASE_TYPE: minor
+
+This release overhauls the health check system in a variety of small ways.
+It adds no new features, but is nevertheless a minor release because it changes
+which tests are likely to fail health checks.
+
+The most noticeable effect is that some tests that used to fail health checks
+will now pass, and some that used to pass will fail. These should all be
+improvements in accuracy. In particular:
+
+* New failures will usually be because they are now taking into account things
+  like use of :func:`~hypothesis.strategies.data()` and
```

Comment:
nit: cross-references don't take parentheses - those are added by Sphinx.  I'm not sure if this would break the formatting, but I'd check that or just remove them.